buildroot-external/overlay/RFD/etc/init.d/S60multimacd

```diff
@@ -109,7 +107,7 @@ start() {
         echo "ERROR: /dev/eq3loop missing"
       fi
     else
-      echo "ERROR: not enough real-time shares (cpu.rt_runtime_us == 0)"
+      echo "ERROR: real-time scheduling failed. Real-time group scheduling CONFIG_RT_GROUP_SCHED enabled, but no run time assigned?"
```

Comment:
Please change to:

`ERROR: real-time scheduling test failed. CONFIG_RT_GROUP_SCHED not set or host kernel unable to utilize sched_setscheduler()`

Reply:
Did you mean "CONFIG_RT_GROUP_SCHED set" or "CONFIG_RT_GROUP_SCHED enabled"?
Otherwise it doesn't make sense to me.